### ISSUE TITLE
Model::findMultipleByIds ignores the order option

### ISSUE THREAD

**Issue by ausi** [contao/core#7940]
Using `Model::findMultipleByIds()` the returned rows are always in the order of the passed IDs. E.g.

```php
\ContentModel::findMultipleByIds(
	array(1, 3, 2, 4), 
	array('order' => 'tl_content.id ASC')
);
```

returns them in the order 1, 3, 2, 4 instead of 1, 2, 3, 4.

**Comment by leofeyer** [comment]
Fixed in 8e9a91e06e7d9f537eb00d3f56f6e96c80e02b5d.

**Comment by ausi** [comment]
Is [Line 856](https://github.com/contao/core/blob/8e9a91e06e7d9f537eb00d3f56f6e96c80e02b5d/system/modules/core/library/Contao/Model.php#L856) correct? Shouldn’t it be something like:

```php
unset($arrUnregistered[array_search($intId, $arrUnregistered)]);
```

**Comment by leofeyer** [comment]
You are right.

**Comment by leofeyer** [comment]
Fixed in 7237954fb53c7752c61f30a31014ad039dd7f57e.
